Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -1615,21 +1611,21 @@ def generate_message_names_implementation(receivers):
     result.append('\n')
     result.append('namespace IPC::Detail {\n')
     result.append('\n')
-    result.append('const MessageDescription messageDescriptions[static_cast<size_t>(MessageName::Count) + 1] = {\n')
+    result.append('const std::array<MessageDescription, static_cast<size_t>(MessageName::Count) + 1> messageDescriptions {\n')
```

Comment:
Then we'd have to omit the first template parameter too. I think it's be OK here though given the code below, it would figure out the type properly. I can likely switch to just `std::array`